Week5/app/src/main/java/com/isabelledionisius/mymovieapp/LoginActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.isabelledionisius.mymovieapp
+
+//import androidx.preference.PreferenceManager
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Gravity
+import android.widget.Button
+import android.widget.EditText
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+
+    // don't need to define them. you can get them directly from xml file
+    lateinit var username: EditText
+    lateinit var password: EditText
+    lateinit var login: Button
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        val prefs = getSharedPreferences("PREFERENCE_NAME", Context.MODE_PRIVATE)
```

Comment:
Avoid having hardcoded values for keys and strings. Also, you could've built this in the App class, and have it exposed there